test-utils/test-utils-core/src/main/java/uk/gov/justice/services/test/utils/core/messaging/MessageConsumerFactory.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.justice.services.test.utils.core.messaging;
+
+import static javax.jms.Session.AUTO_ACKNOWLEDGE;
+
+import javax.jms.Connection;
+import javax.jms.JMSException;
+import javax.jms.MessageConsumer;
+import javax.jms.Session;
+
+import org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory;
+import org.apache.activemq.artemis.jms.client.ActiveMQTopic;
+
+public class MessageConsumerFactory {
+
+    private Session session;
+    private MessageConsumer messageConsumer;
+    private ActiveMQConnectionFactory connectionFactory;
+    private Connection connection;
+
+    public MessageConsumer createAndStart(
```

Comment:
IF we're refactoring this I think we should be a bit clearer about Topic/Queues etc.

Can this be used for Queues? - doesn't look like it.. so maybe we need to consider that..